src/SoundBank.js

```diff
@@ -0,0 +1,84 @@
+const SoundPlayer = require('./GreenPlayer');
+const EffectsChain = require('./EffectsChain');
+
+const ALL_TARGETS = '*';
+
+class SoundBank {
+    constructor (audioEngine) {
+        this.audioEngine = audioEngine;
```

Comment:
I think it'd be best if SoundBank does not know of AudioEngine.

Reply:
The only reason it needs it is for `addSound` and knowing the output node to connect to I think...